td/tracking/templates/tracking/event_form.html

```diff
@@ -53,5 +185,84 @@ <h2 class="panel-title">New Translation Workshop/Event</h2>
 
 {% endblock body%}
 
-{% block extra_script %}
-{% endblock extra_script %}
\ No newline at end of file
+{% block custom_bottom_script %}
```

Comment:
Is this gonna be my own `main.js` that's imported into my `event_form.html`? Can I maybe just link a `.js` file to the HTML?